ppl/zqd/apiserver/filedb.go

```diff
@@ -0,0 +1,201 @@
+package apiserver
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"sync"
+
+	"github.com/brimsec/zq/api"
+	"github.com/brimsec/zq/pkg/iosrc"
+	"github.com/brimsec/zq/zqe"
+)
+
+type FileDb struct {
+	mu   sync.Mutex
+	path iosrc.URI
+}
+
+func CreateFileDb(ctx context.Context, path iosrc.URI, rows []SpaceRow) (*FileDb, error) {
+	ldb := &FileDb{path: path}
```

Comment:
I'm guessing l is for lake, but I wouldn't use it in the names of things in this file that aren't lakes.

Reply:
ah, thanks. Will keep the naming change for a separate PR.